tubesync/sync/migrations/0001_initial.py

```diff
@@ -24,6 +24,7 @@ class Migration(migrations.Migration):
                 ('source_type', models.CharField(choices=[('c', 'YouTube channel'), ('p', 'YouTube playlist')], db_index=True, default='c', help_text='Source type', max_length=1, verbose_name='source type')),
                 ('key', models.CharField(db_index=True, help_text='Source key, such as exact YouTube channel name or playlist ID', max_length=100, unique=True, verbose_name='key')),
                 ('name', models.CharField(db_index=True, help_text='Friendly name for the source, used locally in TubeSync only', max_length=100, unique=True, verbose_name='name')),
+                ('filter_text', models.CharField(db_index=True, help_text='Regex compatible filter string for video titles', max_length=100, verbose_name='filter text')),
```

Comment:
Did you manually add this to the `0001_initial.py` file? Migrations need to be created by `make_migrations` generally. Adding this here will break everyones installations when `migrate` is run on their existing databases.

Reply:
Yes I did add this here, I'm not familiar with this framework but I can see why this would be an issue! Could you point me to info on how to fix this?

Reply:
With Django you just make the adjustments in `models.py` and then run `./manage.py make_migrations`. The `migrations/XXXX_name-here.py` files are created automatically. Once you have a migration you run `./manage.py migrate` and the migrations are checked and applied one at a time in order. You can edit them manually if you need to, but they are in a specific order so if you edit old ones they break the migration chain.

Docs here:

https://docs.djangoproject.com/en/4.2/topics/migrations/

You would need to remove this edit, then use `make_migrations` to create a new migration for this PR.

Reply:
All done, I noticed when I ran makemigrations that it altered two fields from a PR a month ago which I didn't touch in my repo. Hoping that's not an issue as doesn't seem to be from this PR

auto_subtitles:
model.py has a hyphen not in the migration file for that PR

sub_langs:
model.py has a regex validator not in the migration file for that PR